doc/source/reference/arrays.scalars.rst

```diff
@@ -248,7 +248,7 @@ Indexing
 Array scalars can be indexed like 0-dimensional arrays: if *x* is an
 array scalar,
 
-- ``x[()]`` returns a 0-dimensional :class:`ndarray`
+- ``x[...]`` returns a 0-dimensional :class:`ndarray`
```

Comment:
Should keep the first example also, it returns a scalar copy. The second return a view.